interpparser/gpo_cfr.py

```diff
@@ -11,7 +11,7 @@
 from regparser.tree.depth import markers as mtypes
 from regparser.tree.depth import heuristics, rules
 from regparser.tree.depth.derive import derive_depths
-from regparser.tree.gpo_cfr.appendices import appendix_headers
+from regparser.tree.gpo_cfr import appendices
```

Comment:
I attempted to only import the required function (`get_appendix_title`) but my environment complained so I imported the entire module.